155zc.c

```diff
@@ -0,0 +1,62 @@
+typedef struct
+{
+    int capacity;
+    int topofstack;
+    int *array;
+} MinStack;
+
+/** initialize your data structure here. */
+#define EMPTYTOS (-1)
+MinStack* minStackCreate(int maxSize)
+{
+    MinStack *S;
+    S = malloc(sizeof(MinStack)*maxSize);
+    if(S == NULL)
+        return NULL;
+    S->array = malloc(sizeof(int)*maxSize);
+    if(S->array == NULL)
+        return NULL;
+    S->topofstack = EMPTYTOS;
+    S->capacity = maxSize;
+    return S;
+}
+
+void minStackPush(MinStack* obj, int x)
+{
+    if(obj->topofstack == obj->capacity)
```

Comment:
In general, curly braces are not required for single-line statements, but they are allowed if you like them; conditional or loop statements with complex conditions or statements may be more readable with curly braces. Some projects require that an if must always have an accompanying brace.
https://google.github.io/styleguide/cppguide.html#Conditionals